packages/synapse-react-client/src/utils/hooks/useDetectSSOCode.ts

```diff
@@ -80,13 +103,13 @@ export default function useDetectSSOCode(
             // Synapse account not found, send to registration page
             window.location.replace(registerAccountUrl)
           }
-          console.error('Error with Google account association: ', err)
+          console.error('Error with account login: ', err)
           if (onError) {
             onError(err.reason)
           }
         }
 
-        if (state) {
+        if (OAUTH2_PROVIDERS.GOOGLE == provider && state) {
```

Comment:
Got it, this is because we don't support account creation with ORCiD.

Reply:
Exactly